### PR TITLE
doc: add minutes for TypeScript summit

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

**Comment by meyfa** [comment]
Perhaps the second-level headings should use `##` ?

File: meetings/summit-typescript-jul-2022.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+* Wes, check compiler options reference, many different ways to config
+* Zack, two core parts,
+  * print something when test.ts is run with additional info
+  * Lower config than NODE_OPTIONS in terms of what you can do today
+* Andrew, comes down to adding more comprehensive customization hooks
+* Jake - don’t want to get hung up on global list
+* Matteo
+  * Current issue with some proposals - requirement for ESM,world is bigger than that, needs to work for both ESM and CJS
+  * use of NODE_OPTIONS is not ergonomic
+  * loading from Package.json might be a good option
```
Comment (styfle) [inline meetings/summit-typescript-jul-2022.md:71]:
```suggestion
  * loading from package.json might be a good option
```

File: meetings/summit-typescript-jul-2022.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+  configuration into it. - often different configurations for different environments.
+* Wes, expectations that package.json is scoped, maybe you just want a rc file instead.
+* Geoffrey, if file is in scope of package.json, that package.json is used to start/run config,
+  No other package.json take place.
+* Andrew, affects package typically, were most rc files, they are hierarchical, and combined
+* Stephen, tsconfig is not necessary related, defer that to whatever is configured
+* Matteo, we could consider that tsconfig is needed since without that it is sub-par experience
+* Andrew, some talk of mapping from file extensions, some cases where that might not work
+* Michael, more generally or for TypeScript, imports may not have a specifier at all, or
+  Package.json
```
Comment (styfle) [inline meetings/summit-typescript-jul-2022.md:131]:
```suggestion
  package.json
```

**Review by styfle — APPROVED** [review]

**Comment by mhdawson** [comment]
Related PR to document agreement in Node.js core repo (or adjust based on broader collaborator feedback) -https://github.com/nodejs/node/pull/44030

File: meetings/summit-typescript-jul-2022.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+* Matteo
+  * Reason not recommended that can download node and then just run node script.ts. Issue
+    is there are lots of different options
+* Stephen could we just not do the same as ts-node if we don’t use config file
+* Matteo, hook to guide people to setup config file. Deno does not support multiverse of
+  TypeScript, we don’t want to do that.
+* Daniel, in terms of compatibility across versions, does Types as annotations relate in terms of
+  Possibly being defaults?
+* Matteo, whatever happens at runtime level, does not relate, what’s being specified in TC39
+  proposal  dos not overlay, Types a comments is about TypeChecking, not running
```
Comment (sheplu) [inline meetings/summit-typescript-jul-2022.md:86]:
```suggestion
  proposal  does not overlay, Types a comments is about TypeChecking, not running
```

**Comment by sheplu** [comment]
Except for the small typo, LGTM so you can PR in directly :)

**Review by sheplu — APPROVED** [review]

**Pull request merged**